composer.json

```diff
@@ -17,7 +17,8 @@
     "require": {
         "ext-json": "*",
         "php": ">=7.4",
-        "symfony/console": "^5.2"
+        "symfony/console": "^5.3",
+        "symfony/yaml": "^5.3"
```

Comment:
I don't like the idea of introducing a new dependency on this project which doesn't really need to be required for most of the projects using Gacela... I like the idea of having this YamlConfigReader, but I would rather create another repository in the `gacela-project` org, so if someone needs this, they can explicitly require that other repo. But I wouldn't add this dependency to the main repo.